### DISCUSSION TITLE
Dumb Question

### DISCUSSION THREAD

**Discussion by highjohnconquer** [deforum-art/sd-webui-text2video#103] [General]
how do you extend the length of a video? I'm only able to generate a few frames.

**Comment by kabachuha** [comment]
Hi, it's absolutely not a dumb question at all. The number of frames is mostly limited by VRAM, you can extend it by lowering the resolution.

There is also the novel ControlNet-like DiffusionOverDiffusion technique from Microsoft.

I (https://github.com/kabachuha/InfiNet) and the guys from Text2Video-finetuning https://github.com/ExponentialML/Text-To-Video-Finetuning/pull/27 are working to implement it

**Reply by kabachuha** [reply]
If it succeeds, it will allow creating minutes-long coherent videos or even longer

**Reply by highjohnconquer** [reply]
That's awesome thank you
